app/serializers/activitypub/note_serializer.rb

```diff
@@ -1,13 +1,13 @@
 # frozen_string_literal: true
 
 class ActivityPub::NoteSerializer < ActivityPub::Serializer
-  context_extensions :atom_uri, :conversation, :sensitive, :voters_count
+  context_extensions :atom_uri, :conversation, :sensitive, :voters_count, :direct_message
 
   attributes :id, :type, :summary,
              :in_reply_to, :published, :url,
              :attributed_to, :to, :cc, :sensitive,
              :atom_uri, :in_reply_to_atom_uri,
-             :conversation
+             :conversation, :direct_message
```

Comment:
It's nitpicking, but I would prefer if that attribute were only set when the visibility is either `:direct` or `:limited` (`:private` would probably make sense for you if you change how private messages are delivered, but for toots sent from Mastodon/glitch-soc it remains unambiguous at the moment)

Reply:
Would the following be more appropriate?

```
attribute :direct_message, if: :nonpublic?

...

def nonpublic?
  !object.distributable?
end
```

Or should this be expressed another way?

Reply:
This would be great!